blocks/v2-footer/v2-footer.css

```diff
@@ -357,20 +357,33 @@
 }
 
 /* Scroll to top */
-.scroll-to-top {
+
+.v2-scroll-to-top {
   display: none;
   position: fixed;
   bottom: 20px;
```

Comment:
Could we remove the margins and bottom and right to be both 30px?
```
margin: 0;
bottom: 30px;
right: 30px;
```

Reply:
Fixed